cpg-neo4j/src/main/kotlin/de/fraunhofer/aisec/cpg_vis_neo4j/Application.kt

```diff
@@ -165,6 +165,12 @@ class Application : Callable<Int> {
     )
     private var noDefaultPasses: Boolean = false
 
+    @CommandLine.Option(
+        names = ["--custom-pass-list"],
+        description = ["Add custom list of passes (includes --no-default-passes)"]
```

Comment:
This param receives a comma-separated list, right? Could be included in the documentation.